packages/playground/src/weblets/tf_discourse.vue

```diff
@@ -236,6 +236,23 @@ function generatePubKey(): string {
 function updateSSHkeyEnv(selectedKeys: string) {
   selectedSSHKeys.value = selectedKeys;
 }
+watch(
```

Comment:
please use a single watch for both smtp and ipv4 to make code more concise

Reply:
should be fixed.